tests/lambda_functions/va_profile/test_va_profile_integration.py

```diff
@@ -40,6 +61,12 @@ def put_mock(mocker):
     return mocker.patch(f'{LAMBDA_MODULE}.make_PUT_request')
 
 
+@pytest.fixture
+def post_opt_in_confirmation_mock_return(mocker):
```

Comment:
This was created to stub out any new called to send_comp_and_pen_opt_in_confirmation for old tests. The integration test testing the happy path of the new feature IS NOT mocked out like this.